feat(clayui.com): Improve component docs by updating content Pt.2 (#3235)
liferay/clay: packages/clay-progress-bar/README.mdx

</div>
</div>

As long as the process is running, the progress bar grows continuously from 0% to 100%.

> Don't use it to communicate the progress of a user in a certain set of actions. For that you should use [Multi Step Nav](/docs/components/multi-step-nav.html).

Reviewer: ```suggestion
If you need to communicate the progress of a certain set of actions. You should use the [Multi Step Nav](/docs/components/multi-step-nav.html) component.
```
